### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#499] [Esami & Esoneri M-Z]
### Traccia

<img width="935" alt="image" src="https://github.com/user-attachments/assets/b3fc6255-e664-48fe-a6b8-aa4073f21871">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/8feb76c7-eebc-4d5a-8f6b-3a127ee9dbde)
![image](https://github.com/user-attachments/assets/8a4ebd3c-08f0-4fbc-b29b-577903aa02e5)
